src/status_im/ui/screens/browser/bookmarks/views.cljs

```diff
@@ -28,7 +28,7 @@
              :auto-focus          true
              :show-cancel         false
              :label               (i18n/label :t/name)
-             :default-value       name
+             :default-value       @input-name
```

Comment:
Is there any reason why we need the `default-value` prop?
When the component renders, `@input-name` is empty.

Reply:
we set default value when render initialized `(let [input-name (reagent/atom name)]`

Reply:
Okay, I didn't observe that. Approved.